templates/minimal/runtime/Cargo.toml

```diff
@@ -12,47 +12,29 @@ publish = false
 [dependencies]
 codec = { workspace = true }
 scale-info = { workspace = true }
-
-# this is a frame-based runtime, thus importing `frame` with runtime feature enabled.
-frame = { features = [
+polkadot-sdk = { workspace = true, features = [
```

Comment:
This is what I mean, is there really no way to avoid this? esp in our 3 templates, I don't care if e.g. AH that will have pallet-revive are built later like this.

Reply:
I was aware of the tradeoff and was leaning towards the fact that most novice ppl prefer a "one-time" over compilation, in return for never having to touch their toml file again, and letting their IDK figure out which trait comes in from which sp- crate. 

I am not against the runtime feature, but still mildly prefer it to not be the norm. But I'll leave that to the wisdom of the crowd. 

By the way, polkadot-sdk/runtime is meant to be just the things that are needed for building a runtime, minus pallet, it would be more or less identical to frame/runtime.

Reply:
I think it is reasonable to ask people to add a feature when adding a new pallet. It is not something you do regularly after setting up your runtime. The compilation cost you pay all the time. We experienced first hand just now how including `polkadot_sdk` into `revive` balooned the compilation time of that project.